### DISCUSSION TITLE
Issue with onClick in mobile devices

### DISCUSSION THREAD

**Discussion by themachinisst** [replit/kaboom#495] [Q&A]
Right after v2000.2, my game stopped working on mobile devices(android/ios), which was working perfectly before. The game starts with a simple 'play' button, which navigates to game scene when the button is pressed, for which 'OnClick' is used.  Also after latest update I've replaced all deprecated functions, yet its still not detecting any touches on phone.

Can someone help me either to get an older version of kaboom from somewhere, or to understand what exactly could be the issue  !?

**Comment by slmjkdbtl** [comment]
Does the problem only occur on mobile? Are you able to get a minimal reproducible example?

At the mean time, you can just use the previous version, they're always available on npm.
